libs/designsystem/src/lib/components/chart/chart.component.ts

```diff
@@ -148,6 +166,29 @@ export class ChartComponent implements OnInit, OnChanges {
 
           break;
         }
+        case ChartType.COLUMN: {
+          this.options.series = [
+            {
+              type: 'column',
+              data: this.data,
+            },
+          ];
+          (this.options.xAxis as XAxisOptions).categories = this.categories;
+          break;
+        }
+        case ChartType.BAR: {
+          this.options.series = [
+            {
+              type: 'bar',
+              data: this.data,
+            },
+          ];
+          (this.options.xAxis as XAxisOptions).categories = this.categories;
+          break;
+        }
+      }
```

Comment:
Switch case is growing rather big, please refactor content to more readable named methods